plugin/core/sessions.py

```diff
@@ -340,6 +340,15 @@ def on_request_finished_async(self, request_id: int) -> None:
     def on_request_progress(self, request_id: int, params: Dict[str, Any]) -> None:
         ...
 
+    def start_code_lenses_async(self) -> None:
+        ...
+
+    def resolve_visible_code_lenses_async(self) -> None:
+        ...
+
+    def get_resolved_code_lenses_for_region(self, region: sublime.Region) -> Generator[CodeLens, None, None]:
+        ...
+
```

Comment:
I don't think these methods are required to be declared here? You should be able to remove them and mypy won't complain.

Reply:
BTW, you can run `tox` locally to check mypy and flake8.

Reply:
If they're not given a type error is given when you eventually access it down the chain since e.g. `AbstractViewListener` makes use of it and failing to meet that interface leads to a type error.

Reply:
I see. get_resolved_code_lenses_for_region is used in the text command. start_code_lenses_async and resolve_visible_code_lenses_async can go away.